tauv_common/config/thruster_manager.yaml

```diff
@@ -0,0 +1,50 @@
+thruster_manager:
```

Comment:
See below. This file and TAM.yaml should be moved into a subdirectory to indicate what vehicle they're describing. Maybe create a "Models" or "Vehicles" folder in the root of TAUV_common for this.